packages/schematics/angular/ng-new/index.ts

```diff
@@ -60,11 +61,18 @@ export default function (options: NgNewOptions): Rule {
     zoneless: options.zoneless,
   };
 
+  const configOptions: ConfigOptions = {
+    project: options.name,
+    type: ConfigType.Ai,
+    tool: options.aiConfig as unknown as AiTool,
```

Comment:
I'm not sure why I couldn't get the enums to match. It works fine for the packageManager above for example. (Yes I also tried with removing "none" from the enum here in ng-new).